### PR TITLE
Home & create pages

### PR DESCRIPTION
**Pull request by zmbc**

### CODE REVIEW

File: client/src/Components/Create/Create.js [hunk 1]
```diff
@@ -1,12 +1,122 @@
...
+
+                {/*location field*/}
+                <div className="form-group">
+                  <label>Location</label>
+                  <Location />
+                </div>
+
+                {/*schedule field*/}
+                <div className="form-group">
+                  <label>Schedule</label>
```
Comment (juimin) [inline client/src/Components/Create/Create.js]: This label can be removed - seems to be generated by the schedule component

File: client/src/Components/Create/Create.js [hunk 2]
```diff
@@ -1,12 +1,122 @@
...
+
+                {/*email field*/}
+                <div className="form-group">
+                  <label>Email</label>
+                  <input name="email" className="form-control" type="text" placeholder="Type your email" required />
+                </div>
+
+                {/*location field*/}
+                <div className="form-group">
+                  <label>Location</label>
```
Comment (juimin) [inline client/src/Components/Create/Create.js]: The location subview has this title generated already so we can take this out here

File: client/src/Components/Create/Create.js [hunk 3]
```diff
@@ -1,12 +1,122 @@
...
+              <form id="creation-form" onSubmit={this.submit}>
+                {/*title field*/}
+                <div className="form-group">
+                  <label>Meeting Name</label>
+                  <input name="title" className="form-control" type="text" placeholder="Type the meeting's name" required />
+                </div>
+                {/*closeout time field*/}
+                <div className="form-group">
+                  <label>Close-out Time</label>
+                  <input name="closeoutTime" className="form-control" type="text" placeholder="Type the time the meeting should lock in" required />
```
Comment (juimin) [inline client/src/Components/Create/Create.js]: Do we need some kind of input validation or some kind of guideline for how this input should look?

**Review by juimin — CHANGES_REQUESTED** [review]
Some small stuff to take care of. The location picker looks super cool

**Comment by zmbc** [comment]
@JuiMin I think I resolved everything you pointed out.

**Review by juimin — APPROVED** [review]

**Pull request merged**